src/vs/workbench/electron-browser/commands.ts

```diff
@@ -423,3 +424,16 @@ export function registerCommands(): void {
 		});
 	});
 }
+
+namespace GlobalQuickOpenCommand {
+	export const ID = 'vscode.quickOpen';
```

Comment:
@cristianhosu shouldn't this be the same ID as the action we already have and not introduce a new command ID? And the action we had so far should use this command too.